tempo/serve/utils.py

```diff
@@ -77,7 +77,7 @@ def __call__(self, *args, **kwargs) -> Any:
 
 def _bind_tempo_interface(artifact: Any, model: BaseModel) -> Any:
     setattr(artifact, "request", model.request)
-    setattr(artifact, "remote", model.remote)
+    setattr(artifact, "remote", model.predict)
```

Comment:
Shoudl this not be `predict` -> `predict`?

Reply:
yes

Reply:
actually no. I wanted to leave as is as none of the example would use this and eventually might be removed.

Reply:
Also more likely `predict` will clash with existing user method

Reply:
Ah oh i see... yeah this is the sort of stuf that gets confusing